.circleci/config.yml

```diff
@@ -879,6 +905,10 @@ workflows:
       - t_osx_cli: *workflow_osx
       - t_osx_soltest: *workflow_osx
 
+      # ArchLinux build and tests
+      - b_archlinux: *workflow_trigger_on_tags
+      - t_archlinux_soltest: *workflow_archlinux
```

Comment:
Maybe it would be better to "inline" `workflow_archlinux` here.

Reply:
Ah no, but can you replace `test_archlinux` by `t_archlinux_soltest`?

Reply:
Sure